cmake/alpakaCommon.cmake

```diff
@@ -557,6 +564,13 @@ if(alpaka_ACC_GPU_HIP_ENABLE)
             alpaka_set_compiler_options(DEVICE target alpaka "-ffast-math")
         endif()
 
+        if(alpaka_RELOCATABLE_DEVICE_CODE)
+            alpaka_set_compiler_options(DEVICE target alpaka "-fgpu-rdc")
+            target_link_options(alpaka INTERFACE "-fgpu-rdc" "--hip-link")
+        elseif(NOT alpaka_RELOCATABLE_DEVICE_CODE)
+            alpaka_set_compiler_options(DEVICE target alpaka "-fno-gpu-rdc")
+        endif()
+
```

Comment:
@psychocoderHPC Do you know whether there are any missing options here? All ROCm CI Runners are failing with an ICE sooner or later.